job-server/src/spark.jobserver/AkkaClusterSupervisorActor.scala

```diff
@@ -205,10 +205,15 @@ class AkkaClusterSupervisorActor(daoActor: ActorRef) extends InstrumentedActor {
           return
       }
 
+    var driverMemory = "0"
+    if (contextConfig.hasPath("spark.driver") && contextConfig.hasPath("spark.driver.memory")) {
```

Comment:
Ficus is fantastic btw, takes out a lot of boilerplate code for checking the existence of keys first and retrieving them when you don't know their types ahead of time.